ordinary_data/views/drop_views.sql

```diff
@@ -1,53 +1,53 @@
 
 /* schematic */
-drop view if exists qwat_od.vw_pipe_schema_error;
-drop view if exists qwat_od.vw_pipe_schema;
-drop view if exists qwat_od.vw_pipe_schema_merged;
-drop view if exists qwat_od.vw_pipe_schema_visibleitems;
-drop view if exists qwat_od.vw_pressurezone_simplified;
+drop view if exists qwat_od.vw_pipe_schema_error cascade;
```

Comment:
Please don't write `CASCADE`, there's good chance to erase views from the user. To me it's hard requirement that we never write cascade in a delta or in the view scripts.

Reply:
You're right. Let's see how we can do otherwise.